packages/components/src/Datepicker/Datepicker.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { ReactElement, cloneElement, useState } from "react";
+import ReactDatePicker from "react-datepicker";
+import { XOR } from "ts-xor";
+import { Button } from "../Button";
+import { strFormatDate } from "../FormatDate";
+
+/**
+ * Disabling no-internal-modules here because we need
+ * to reach into the package to get the css file.
+ */
+// eslint-disable-next-line import/no-internal-modules
+import "react-datepicker/dist/react-datepicker.css";
+
+interface DatePickerReturnedDates {
+  readonly formatted: string;
+  readonly raw: Date;
+}
```

Comment:
@joelwarrington-jobber This is what i was thinking for the return value, this way we have a "readable" date and a raw date that we can do things like store in DBs

Reply:
I like it!